### PR TITLE
Refactor of Scripts.js

### LINKED ISSUE #57: Rename and Reorganize scripts.js
Reorganize and rename for readability.

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
### Description
Renaming and reorganizing global variables and functions for readability. 
### Related Issue
Revolves #57
### Motivation and Context
`scripts.js` was deeply unorganized, and various names needed to be redone.
### Types of changes
- [x] Refactor
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -9,26 +9,27 @@ import './css/styles.scss';
 import User from './user';
 import Recipe from './recipe';
 
+let main = document.querySelector("main");
+let fullRecipeInfo = document.querySelector(".recipe-instructions");
+let tagList = document.querySelector(".tag-list");
+let searchForm = document.querySelector("#search");
+let searchInput = document.querySelector("#search-input");
```
Comment (MichaelEWalker87) [inline src/scripts.js:16]: looks good great work

File: src/scripts.js [hunk 2]
```diff
@@ -9,26 +9,27 @@ import './css/styles.scss';
...
-let recipes = [];
 let savedRecipesBtn = document.querySelector(".saved-recipes-btn");
 let searchBtn = document.querySelector(".search-btn");
-let searchForm = document.querySelector("#search");
-let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
-let tagList = document.querySelector(".tag-list");
+let menuOpen = false;
+let pantryInfo = [];
+let recipes = [];
```
Comment (MichaelEWalker87) [inline src/scripts.js:25]: do we want these to be global with this change with the data set come from fetch

File: src/scripts.js [hunk 3]
```diff
@@ -57,23 +58,29 @@ function createRecipeObject(recipes) {
   return newRecipeObjects
 }
 
+function capitalize(words) {
+  return words.split(" ").map(word => {
+    return word.charAt(0).toUpperCase() + word.slice(1);
+  }).join(" ");
+}
+
```
Comment (MichaelEWalker87) [inline src/scripts.js:66]: Nice word

**Review by MichaelEWalker87 — APPROVED** [review]
I think that this cleans up the scripts file and make it much more readable great job

File: src/scripts.js (hunk 2 above)
Comment (Estaffieri) [reply src/scripts.js:25]: I think the fetch stuff might absorb some of these but, once we merge everything we can figure that step out. I know that they will handle much of the lift for rendering.

File: src/scripts.js [hunk 4]
```diff
@@ -171,6 +172,14 @@ function addToMyRecipes() {
   } else if (isDescendant(event.target.closest(".recipe-card"), event.target)) {
     openRecipeInfo(event);
   }
+} //this could be refactored. look at seperating the functionality of changing the apple image into its own function, might make things more srp. add it to the wishlist if we get stuff finished up! would have to add some functionality to the user.js class as well to get this working and seperated. 
+
+function addToMyRecipes() {
+
+}
+
+function toggleApples() {
```
Comment (Estaffieri) [inline src/scripts.js:181]: Best function name! Toggle them apples!

**Review by Estaffieri — APPROVED** [review]
Thank you for making these refactored changes! This makes the style more consistent and really cleans up the naming conventions, and organizing how the data flows.

**Pull request merged**